examples/fxa-geodb-example.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+var path = require('path');
+
+(function() {
+  'use strict';
```

Comment:
add an inline note re: how to include the module via npm before the require statement on line 10.